monai/utils/module.py

```diff
@@ -220,6 +228,14 @@ def instantiate(path: str, **kwargs):
     if component is None:
         raise ModuleNotFoundError(f"Cannot locate class or function path: '{path}'.")
     try:
+        if kwargs.pop("_debug_", False) or run_debug:
+            warnings.warn(
+                f"\n\npdb: instantiating component={component}\n"
+                f"See also Debugger commands documentation: https://docs.python.org/3/library/pdb.html\n"
+            )
+            import pdb
+
+            pdb.set_trace()
```

Comment:
Is it possible to add some unit tests to cover the behavior here?
I didn't test `pdb` locally, I think it's an interactive operation?

Thanks.